### ISSUE TITLE
(Request) Shuttle OMS Engine Power Increase

### ISSUE THREAD

**Issue by KamikazeKricket** [shadowmage45/SSTULabs#700]
So, I've been playing around a lot recently with the shuttle, and its showing that its pretty hard to do a lot with it. It seems like I don't even have enough fuel to preform a simple rendezvous with a low orbit station and return home. 

I get that things are suppose to be more realistic, and the OMS engines weren't that strong, but it also took the Apollo Service Module 4 1/2 minutes to lower the apoapsis to 170 miles at a periapsis of 69 miles during the missions. 

For gameplay purposes alone, I think the OMS engines on the shuttle need to be bumped up to at least twice their strength at the same fuel consumption rate.

**Comment by blowfishpro** [comment]
All I can say is physics warp is you friend...

**Comment by Jimbodiah** [comment]
gheghe.... I have no patience either. Timewarp is always running when I do maneuvres. :)

**Comment by Theysen** [comment]
it's the sweet balancing point between realistic burn times and stock scale. I agree, also the Orion SM has somewhat of a 14 minute burn time and while thats totally fine in RSS / RO you're doing what it feels like half a orbit while circularizing around the Mun burning your engines. But balancing against stock is hard. and tedious and way too much trial and error. 

The optimal case would be a set of MM configs which activate depending on your installed scale balancing masses, thrusts and ISP's of the engines. 

It's the same deal with the line of RL-10s, while fine in real scale you often require a lot more engines for circularizing in time in stock scale just because you dont have enough time. 

tl;dr stock scale is the cause of all evil

**Comment by Jimbodiah** [comment]
I have a patch that increases thrust on the orion by 3x. Check my patch pack at spacedock.
